src/app/pages/loading-waiting-states/contextual-spinners/contextual-spinners.component.scss

```diff
@@ -0,0 +1,53 @@
+@import '~@pxblue/colors/palette.scss';
+
+.extended-fab-button {
+    width: 180px;
+    height: 56px;
+    padding: 0 26px;
```

Comment:
The padding here is a little odd, but I can see why you did it that way. I think it's fine for now.